client/extensions/woocommerce/app/dashboard/widgets/stats-widget/index.js

```diff
@@ -0,0 +1,363 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import React, { Component, Fragment } from 'react';
+import { bindActionCreators } from 'redux';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { moment, localize } from 'i18n-calypso';
+import { sortBy, find } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { dashboardListLimit } from 'woocommerce/app/store-stats/constants';
+import DashboardWidget from 'woocommerce/components/dashboard-widget';
+import { getLink } from 'woocommerce/lib/nav-utils';
+import { getPreference } from 'state/preferences/selectors';
+import { getSelectedSiteWithFallback } from 'woocommerce/state/sites/selectors';
+import { getSiteStatsNormalizedData } from 'state/stats/lists/selectors';
+import { getQueries } from './queries';
+import {
+	getUnitPeriod,
+	getStartPeriod,
+	getDelta,
+	getDeltaFromData,
+	getEndPeriod,
+	getConversionRateData,
+} from 'woocommerce/app/store-stats/utils';
+import List from './list';
+import QueryPreferences from 'components/data/query-preferences';
+import QuerySiteStats from 'components/data/query-site-stats';
+import { savePreference } from 'state/preferences/actions';
+import SelectDropdown from 'components/select-dropdown';
+import Stat from './stat';
+
+class StatsWidget extends Component {
+	static propTypes = {
+		site: PropTypes.shape( {
+			name: PropTypes.string.isRequired,
+			slug: PropTypes.string.isRequired,
+		} ),
+		unit: PropTypes.string,
+		queries: PropTypes.object,
+		orderData: PropTypes.object,
+		referrerData: PropTypes.array,
+		topEarnersData: PropTypes.array,
+		visitorData: PropTypes.array,
+		productData: PropTypes.array,
+		saveDashboardUnit: PropTypes.func,
+	};
+
+	handleTimePeriodChange = option => {
+		const { saveDashboardUnit } = this.props;
+		saveDashboardUnit( option.value );
+	};
+
+	dateForDisplay = () => {
+		const { translate, unit } = this.props;
+
+		const localizedDate = moment( moment().format( 'YYYY-MM-DD' ) );
```

Comment:
The `localize` HOC also gives you `moment`, so you can grab this from props just like `translate`